testing/u-boot-s5p6818/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor:
+# Maintainer:
+pkgname=u-boot-s5p6818
+pkgver=1.2
+pkgrel=0
+pkgdesc="u-boot for s5p6818"
+url="https://github.com/rafaello7/u-boot-nanopi-m3"
+arch="aarch64"
+license="BSD-2-Clause"
+depends=""
+depends_dev=""
+makedepends="$depends_dev bsd-compat-headers openssl-dev"
```

Comment:
depends_dev will be removed when empty variables can be removed and this reference to it can be removed

Reply:
Removed. Thank you!